src/components/navbar.js

```diff
@@ -22,17 +22,19 @@ function Navbar(props) {
           transition: 'all 0.3s ease-in-out',
         }}
       >
-        <img
-          className="fadeInUp logo"
-          alt="India COVID-19 Tracker"
-          src="/icon.png"
-          style={{
-            animationDelay: '0.0s',
-            width: view === 'Clusters' ? '1.5rem' : '',
-            height: view === 'Clusters' ? '1.5rem' : '',
-            transition: 'all 0.3s ease-in-out',
-          }}
-        />
+        <Link to="/" className="img-link">
```

Comment:
@saberprashant : A suggestion , there is already css for .Navbar img.. Possible to add css there instead of creating new className

Reply:
Hi @Rajesh-Narayanappa87, I actually used it for the link tag, but I'll definitely try to incorporate the same in the **img** class.

Reply:
@Rajesh-Narayanappa87, I made the changes, and removed that extra **img-link** class, plus, I've added a little style to existing **img** class, to give it a good look when someone hovers over it.

Reply:
@JunaidBabu , please review it if possible.
PS: I didn't know that I have to get it assigned first, but I think I was the first one to start working on this one.